CONTRIBUTING.md

```diff
@@ -139,7 +139,16 @@ A guide on commonly used labels added to issues and pull requests:
 
 ## Testing Your Code
 
-- You can test your code locally along with your changes using: `yarn storybook` or `npm run storybook`. This will open the storybook in your browser .
+- You can test your code locally along with your changes using: `yarn storybook` or `npm run storybook`. This will open the storybook in your browser.
```

Comment:
```suggestion
**Manual Tests**
- You can test your code locally along with your changes using: `yarn storybook` or `npm run storybook`. This will open the storybook in your browser.
```